ftw/testbrowser/core.py

```diff
@@ -316,7 +317,10 @@ def webdav(self, method, url_or_object=None, data=None, view=None,
         """
         self._verify_setup()
         url = self._normalize_url(url_or_object, view=view)
-        driver = self.get_driver(LIB_REQUESTS)
+        driver = self.get_driver()
+        if not driver.WEBDAV_SUPPORT:
```

Comment:
i would not use an all upper-case name here, i think this is not a constant but a class-level configuration attribute. what do you think?